docs/develop/topics/administration_panel.md

```diff
@@ -0,0 +1,432 @@
+        "search_pattern": {"order": 4, "text": "Search query"},
+        "created": {"order": 5},
+        "updated": {"order": 6},
+    }
+```
+
+For the full attributes list and description visit [reference docs](../../reference/administration_reference.md)
+
+
+#### Details view
+
+![Details view layout](./img/administration/administration_details_view.png)
+By default, this view displays the details of a selected resource. It can be configured to display/hide a set of fields.
+
+It can be extended or completely overridden by a custom view.
+
+For OAI-PMH sets, as seen in the figure, this view uses a custom jinja template and `React` to render a second table that displays OAI-PMH Sets links.
+
+More information on template override and custom react components will be detailed later in this guide.
+
+##### Usage
+
+This is an example of DetailView configuration for OAI-PMH set resource.
+
+```python
+class OaiPmhDetailView(AdminResourceDetailView):
+    """Configuration for OAI-PMH sets detail view."""
+
+    url = "/oai-pmh/<pid_value>"
+    api_endpoint = "/oaipmh/sets"
+    search_request_headers = {"Accept": "application/json"}
+    name = "OAI-PMH details"
+    resource_config = "oaipmh_server_resource"
+    title = "OAI-PMH Details"
+
+    template = "invenio_rdm_records/oai-details.html"
+    display_delete = True
+    display_edit = True
+
+    list_view_name = "OAI-PMH"
+    pid_path = "id"
+
+    item_field_list = {
+        "name": {"text": "Set name", "order": 1},
+        "spec": {"text": "Set spec", "order": 2},
+        "search_pattern": {"text": "Search query", "order": 3},
+        "created": {"text": "Created", "order": 4},
+        "updated": {"text": "Updated", "order": 5},
+    }
+```
+
+For the full attributes list and description visit [reference docs](../../reference/administration_reference.md)
+
+#### Views registration
+
+In order to render views in `invenio-administration`, they must be registered from the module on which they are implemented.
+
+A typical flow of actions to register views for a module is as follows:
+
+1. Create views in the module.
+2. Configure views to match your needs.
+3. Register views in `invenio-administration` entry point group.
+
+In OAI-PMH sets example, views are registered in `invenio-administration` as individual entry points from `invenio-rdm-records`.
+
+```bash
+vim invenio-rdm-records/setup.cfg
+```
+
+```ini
+[options.entry_points]
+invenio_administration.views =
+    invenio_rdm_records_oai_list = invenio_rdm_records.administration.views.oai:OaiPmhListView
+    invenio_rdm_records_oai_edit = invenio_rdm_records.administration.views.oai:OaiPmhEditView
+    invenio_rdm_records_oai_create = invenio_rdm_records.administration.views.oai:OaiPmhCreateView
+    invenio_rdm_records_details = invenio_rdm_records.administration.views.oai:OaiPmhDetailView
+```
+
+### Create custom view
+
+A custom view can be created by inheriting directly from `AdminView`.
+#### Usage
+
+```python
```

Comment:
I would add:
`In your module, create a new folder (anchor link to the folder structure) and a the following filename.py file:`